lita_config.rb

```diff
@@ -53,7 +53,7 @@
   config.handlers.static_meme.mapping = {
     /^where('| i)s #{config.robot.name}[?]?$/i => "There is no #{config.robot.name}. There is only Zuul.",
     /^thank(s| ?you)\s#{config.robot.name}/i => "At your service.",
-    /top .* the mornin/i => "And the rest of the day to yourself.",
+    /top o.* the mornin/i => "And the rest of the day to yourself.",
```

Comment:
Good catch. I wonder if `\btop o? the morning/i` would be better? The `\b` looks for a "boundary", so it wouldn't match "stop". And I guess `o?` would be fine, because it matches "o", "oh", "o'", etc.

Of course, how silly is this outright? 😆